src/core/server/metrics/metrics_service.ts

```diff
@@ -71,6 +74,8 @@ export class MetricsService
   private async refreshMetrics() {
     this.logger.debug('Refreshing metrics');
     const metrics = await this.metricsCollector!.collect();
+    const { message, ...rest } = getEcsOpsMetricsLog(metrics);
+    this.opsMetricsLogger.debug(message!, rest);
```

Comment:
Optional: according to the single responsibility principle, it shouldn't belong to `refreshMetrics` but should subscribe to `this.metrics$`. Although, it's okay to keep it here as long as `refreshMetrics` body is small.